[2025-03-15 12:21:21,657: INFO: main: Welcome to our custom logging data science]
[2025-03-15 14:24:17,247: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 14:24:17,253: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 14:25:03,035: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 14:25:03,040: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 14:25:32,029: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 14:25:32,032: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 14:25:32,033: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 14:25:32,039: INFO: common: created directory at: artifacts]
[2025-03-15 14:25:32,043: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-15 14:27:13,705: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 14:27:13,710: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 14:27:13,715: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 14:27:13,717: INFO: common: created directory at: artifacts]
[2025-03-15 14:27:13,721: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-15 14:28:14,764: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 14:28:14,768: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 14:28:14,772: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 14:28:14,772: INFO: common: created directory at: artifacts]
[2025-03-15 14:28:14,779: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-15 14:29:39,959: INFO: 3182789448: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 4EFE:216C75:626D7:111BA8:67D5417B
Accept-Ranges: bytes
Date: Sat, 15 Mar 2025 08:59:39 GMT
Via: 1.1 varnish
X-Served-By: cache-hyd1100027-HYD
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1742029179.303017,VS0,VE265
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: c690d4eec92bd241789386b4048bba1c8a192036
Expires: Sat, 15 Mar 2025 09:04:39 GMT
Source-Age: 0

]
[2025-03-15 14:33:15,346: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 14:33:15,352: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 14:33:15,359: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 14:33:15,362: INFO: common: created directory at: artifacts]
[2025-03-15 14:33:15,362: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-15 14:33:15,367: INFO: 3182789448: File already exists]
[2025-03-15 17:56:31,595: INFO: main: >>>>> stage Data Ingestion Stage started <<<<]
[2025-03-15 17:56:31,599: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 17:56:31,599: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 17:56:31,601: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 17:56:31,602: ERROR: main: "'ConfigBox' object has no attribute 'artifacts_root'"]
Traceback (most recent call last):
  File "box\\box.py", line 594, in box.box.Box.__getitem__
KeyError: 'artifacts_root'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\box.py", line 633, in box.box.Box.__getattr__
  File "box\\box.py", line 621, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'artifacts_root'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 635, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'artifacts_root'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\config_box.py", line 29, in box.config_box.ConfigBox.__getattr__
  File "box\\box.py", line 649, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'artifacts_root'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 594, in box.box.Box.__getitem__
KeyError: 'artifacts_root'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\box.py", line 633, in box.box.Box.__getattr__
  File "box\\box.py", line 621, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'artifacts_root'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 635, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'artifacts_root'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "E:\DataScienceProject1\main.py", line 9, in <module>
    data_ingestion.initiate_data_ingestion()
  File "E:\DataScienceProject1\src\datascience\pipeline\data_ingestion_pipeline.py", line 12, in initiate_data_ingestion
    config=ConfigurationManager()
  File "E:\DataScienceProject1\src\datascience\config\configuration.py", line 15, in __init__
    create_directories([self.config.artifacts_root])
  File "box\\config_box.py", line 31, in box.config_box.ConfigBox.__getattr__
  File "box\\box.py", line 649, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'artifacts_root'"
[2025-03-15 18:03:20,016: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-03-15 18:03:20,019: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 18:03:20,019: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 18:03:20,022: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 18:03:20,022: ERROR: main: "'ConfigBox' object has no attribute 'artifacts_root'"]
Traceback (most recent call last):
  File "box\\box.py", line 594, in box.box.Box.__getitem__
KeyError: 'artifacts_root'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\box.py", line 633, in box.box.Box.__getattr__
  File "box\\box.py", line 621, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'artifacts_root'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 635, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'artifacts_root'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\config_box.py", line 29, in box.config_box.ConfigBox.__getattr__
  File "box\\box.py", line 649, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'artifacts_root'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 594, in box.box.Box.__getitem__
KeyError: 'artifacts_root'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\box.py", line 633, in box.box.Box.__getattr__
  File "box\\box.py", line 621, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'artifacts_root'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 635, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'artifacts_root'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "E:\DataScienceProject1\main.py", line 9, in <module>
    data_ingestion.initiate_data_ingestion()
  File "E:\DataScienceProject1\src\datascience\pipeline\data_ingestion_pipeline.py", line 12, in initiate_data_ingestion
    config=ConfigurationManager()
  File "E:\DataScienceProject1\src\datascience\config\configuration.py", line 15, in __init__
    create_directories([self.config.artifacts_root])
  File "box\\config_box.py", line 31, in box.config_box.ConfigBox.__getattr__
  File "box\\box.py", line 649, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'artifacts_root'"
[2025-03-15 18:32:58,638: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-03-15 18:32:58,643: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 18:32:58,643: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 18:32:58,643: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 18:32:58,643: INFO: common: created directory at: artifacts]
[2025-03-15 18:32:58,643: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-15 18:34:24,131: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 7CE1:937E5:A3815:1A8C35:67D57AD6
Accept-Ranges: bytes
Date: Sat, 15 Mar 2025 13:04:23 GMT
Via: 1.1 varnish
X-Served-By: cache-hyd1100024-HYD
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1742043863.797567,VS0,VE562
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 6e30ac08d40f83541a093a8efd7364c832a25552
Expires: Sat, 15 Mar 2025 13:09:23 GMT
Source-Age: 0

]
[2025-03-15 18:34:24,135: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-03-15 19:47:27,590: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 19:47:27,591: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 19:47:27,597: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 19:49:58,578: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 19:49:58,582: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 19:49:58,586: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 19:49:58,593: INFO: common: created directory at: artifacts]
[2025-03-15 19:49:58,595: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-15 20:00:32,686: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 20:00:32,691: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 20:00:32,696: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 20:00:32,699: INFO: common: created directory at: artifacts]
[2025-03-15 20:00:32,704: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-15 20:00:44,558: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 20:00:44,561: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 20:00:44,566: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 20:00:44,572: INFO: common: created directory at: artifacts]
[2025-03-15 20:00:44,574: INFO: common: created directory at: artifacts/data_validation]
[2025-03-15 20:32:16,438: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-03-15 20:32:16,448: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 20:32:16,449: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 20:32:16,451: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 20:32:16,451: INFO: common: created directory at: artifacts]
[2025-03-15 20:32:16,451: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-15 20:32:16,451: INFO: data_ingestion: File already exists]
[2025-03-15 20:32:16,451: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-03-15 20:32:16,451: INFO: main: >>>> stage Data Validation Stage started <<<<]
[2025-03-15 20:32:16,463: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 20:32:16,465: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 20:32:16,470: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 20:32:16,473: INFO: common: created directory at: artifacts]
[2025-03-15 20:32:16,473: INFO: common: created directory at: artifacts/data_validation]
[2025-03-15 20:32:16,504: INFO: main: >>>> stage Data Validation Stage completed <<<<

x======x]
[2025-03-15 21:46:31,167: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 21:46:31,170: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 21:46:31,177: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 21:46:31,178: INFO: common: created directory at: artifacts]
[2025-03-15 21:46:31,185: INFO: common: created directory at: artifacts/data_transformation]
[2025-03-15 21:46:31,239: INFO: 3713794891: Splitted data into training and test sets]
[2025-03-15 21:46:31,243: INFO: 3713794891: (1199, 12)]
[2025-03-15 21:46:31,244: INFO: 3713794891: (400, 12)]
[2025-03-15 22:16:31,907: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-03-15 22:16:31,909: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 22:16:31,909: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 22:16:31,914: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 22:16:31,914: INFO: common: created directory at: artifacts]
[2025-03-15 22:16:31,914: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-15 22:16:31,914: INFO: data_ingestion: File already exists]
[2025-03-15 22:16:31,917: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-03-15 22:16:31,917: INFO: main: >>>> stage Data Validation Stage started <<<<]
[2025-03-15 22:16:31,920: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 22:16:31,924: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 22:16:31,927: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 22:16:31,936: INFO: common: created directory at: artifacts]
[2025-03-15 22:16:31,937: INFO: common: created directory at: artifacts/data_validation]
[2025-03-15 22:16:31,951: INFO: main: >>>> stage Data Validation Stage completed <<<<

x======x]
[2025-03-15 22:16:31,951: INFO: main: >>> stage Data Transformation stage started <<<]
[2025-03-15 22:16:31,951: INFO: main: >>> stage Data Transformation stage completed <<<

x=====x]
[2025-03-15 22:18:11,003: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-03-15 22:18:11,012: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 22:18:11,012: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 22:18:11,017: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 22:18:11,017: INFO: common: created directory at: artifacts]
[2025-03-15 22:18:11,017: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-15 22:18:11,017: INFO: data_ingestion: File already exists]
[2025-03-15 22:18:11,022: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-03-15 22:18:11,022: INFO: main: >>>> stage Data Validation Stage started <<<<]
[2025-03-15 22:18:11,026: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 22:18:11,027: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 22:18:11,031: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 22:18:11,031: INFO: common: created directory at: artifacts]
[2025-03-15 22:18:11,036: INFO: common: created directory at: artifacts/data_validation]
[2025-03-15 22:18:11,052: INFO: main: >>>> stage Data Validation Stage completed <<<<

x======x]
[2025-03-15 22:18:11,054: INFO: main: >>> stage Data Transformation stage started <<<]
[2025-03-15 22:18:11,055: INFO: main: >>> stage Data Transformation stage completed <<<

x=====x]
[2025-03-15 22:19:12,485: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-03-15 22:19:12,488: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 22:19:12,488: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 22:19:12,494: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 22:19:12,496: INFO: common: created directory at: artifacts]
[2025-03-15 22:19:12,496: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-15 22:19:12,496: INFO: data_ingestion: File already exists]
[2025-03-15 22:19:12,509: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-03-15 22:19:12,509: INFO: main: >>>> stage Data Validation Stage started <<<<]
[2025-03-15 22:19:12,511: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 22:19:12,512: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 22:19:12,515: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 22:19:12,515: INFO: common: created directory at: artifacts]
[2025-03-15 22:19:12,515: INFO: common: created directory at: artifacts/data_validation]
[2025-03-15 22:19:12,531: INFO: main: >>>> stage Data Validation Stage completed <<<<

x======x]
[2025-03-15 22:19:12,531: INFO: main: >>> stage Data Transformation stage started <<<]
[2025-03-15 22:19:12,531: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 22:19:12,537: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 22:19:12,540: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 22:19:12,540: INFO: common: created directory at: artifacts]
[2025-03-15 22:19:12,540: INFO: common: created directory at: artifacts/data_transformation]
[2025-03-15 22:19:12,567: INFO: data_transformation: Splitted data into training and test sets]
[2025-03-15 22:19:12,569: INFO: data_transformation: (1199, 12)]
[2025-03-15 22:19:12,570: INFO: data_transformation: (400, 12)]
[2025-03-15 22:19:12,570: INFO: main: >>> stage Data Transformation stage completed <<<

x=====x]
[2025-03-15 23:38:33,060: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 23:38:33,066: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 23:38:33,076: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 23:38:33,080: INFO: common: created directory at: artifacts]
[2025-03-15 23:40:47,809: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 23:40:47,815: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 23:40:47,819: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 23:40:47,824: INFO: common: created directory at: artifacts]
[2025-03-15 23:41:49,893: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 23:41:49,896: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 23:41:49,903: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 23:41:49,907: INFO: common: created directory at: artifacts]
[2025-03-15 23:41:49,909: INFO: common: created directory at: artifacts/model_trainer]
[2025-03-15 23:57:51,214: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-03-15 23:57:51,217: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 23:57:51,217: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 23:57:51,221: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 23:57:51,221: INFO: common: created directory at: artifacts]
[2025-03-15 23:57:51,221: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-15 23:57:51,221: INFO: data_ingestion: File already exists]
[2025-03-15 23:57:51,221: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-03-15 23:57:51,221: INFO: main: >>>> stage Data Validation Stage started <<<<]
[2025-03-15 23:57:51,229: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 23:57:51,231: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 23:57:51,231: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 23:57:51,231: INFO: common: created directory at: artifacts]
[2025-03-15 23:57:51,231: INFO: common: created directory at: artifacts/data_validation]
[2025-03-15 23:57:51,252: INFO: main: >>>> stage Data Validation Stage completed <<<<

x======x]
[2025-03-15 23:57:51,252: INFO: main: >>> stage Data Transformation stage started <<<]
[2025-03-15 23:57:51,260: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 23:57:51,263: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 23:57:51,265: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 23:57:51,267: INFO: common: created directory at: artifacts]
[2025-03-15 23:57:51,267: INFO: common: created directory at: artifacts/data_transformation]
[2025-03-15 23:57:51,291: INFO: data_transformation: Splitted data into training and test sets]
[2025-03-15 23:57:51,291: INFO: data_transformation: (1199, 12)]
[2025-03-15 23:57:51,291: INFO: data_transformation: (400, 12)]
[2025-03-15 23:57:51,291: INFO: main: >>> stage Data Transformation stage completed <<<

x=====x]
[2025-03-15 23:57:51,291: INFO: main: >>> stage Model Trainer Stage started <<<]
[2025-03-15 23:57:51,298: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 23:57:51,301: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 23:57:51,304: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 23:57:51,306: INFO: common: created directory at: artifacts]
[2025-03-15 23:57:51,308: INFO: common: created directory at: artifacts/model_trainer]
[2025-03-15 23:57:51,322: INFO: main: >>> stage Model Trainer Stage completed <<<

x=====x]
[2025-03-15 23:58:19,400: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-03-15 23:58:19,402: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 23:58:19,402: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 23:58:19,407: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 23:58:19,407: INFO: common: created directory at: artifacts]
[2025-03-15 23:58:19,410: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-15 23:58:19,411: INFO: data_ingestion: File already exists]
[2025-03-15 23:58:19,415: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-03-15 23:58:19,416: INFO: main: >>>> stage Data Validation Stage started <<<<]
[2025-03-15 23:58:19,419: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 23:58:19,424: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 23:58:19,427: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 23:58:19,427: INFO: common: created directory at: artifacts]
[2025-03-15 23:58:19,429: INFO: common: created directory at: artifacts/data_validation]
[2025-03-15 23:58:19,441: INFO: main: >>>> stage Data Validation Stage completed <<<<

x======x]
[2025-03-15 23:58:19,441: INFO: main: >>> stage Data Transformation stage started <<<]
[2025-03-15 23:58:19,445: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 23:58:19,445: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 23:58:19,449: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 23:58:19,449: INFO: common: created directory at: artifacts]
[2025-03-15 23:58:19,452: INFO: common: created directory at: artifacts/data_transformation]
[2025-03-15 23:58:19,476: INFO: data_transformation: Splitted data into training and test sets]
[2025-03-15 23:58:19,476: INFO: data_transformation: (1199, 12)]
[2025-03-15 23:58:19,476: INFO: data_transformation: (400, 12)]
[2025-03-15 23:58:19,479: INFO: main: >>> stage Data Transformation stage completed <<<

x=====x]
[2025-03-15 23:58:19,479: INFO: main: >>> stage Model Trainer Stage started <<<]
[2025-03-15 23:58:19,483: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-15 23:58:19,485: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-15 23:58:19,488: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-15 23:58:19,488: INFO: common: created directory at: artifacts]
[2025-03-15 23:58:19,491: INFO: common: created directory at: artifacts/model_trainer]
[2025-03-15 23:58:19,506: INFO: main: >>> stage Model Trainer Stage completed <<<

x=====x]
[2025-03-16 18:44:51,871: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 18:44:51,910: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 18:44:51,944: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 18:44:51,953: INFO: common: created directory at: artifacts]
[2025-03-16 18:44:51,961: INFO: common: created directory at: artifacts/model_evaluation]
[2025-03-16 18:45:25,957: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 18:45:25,961: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 18:45:25,968: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 18:45:25,973: INFO: common: created directory at: artifacts]
[2025-03-16 18:45:25,973: INFO: common: created directory at: artifacts/model_evaluation]
[2025-03-16 18:45:26,449: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json]
[2025-03-16 19:16:25,177: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-03-16 19:16:25,180: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 19:16:25,180: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 19:16:25,180: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 19:16:25,188: INFO: common: created directory at: artifacts]
[2025-03-16 19:16:25,188: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-16 19:16:25,188: INFO: data_ingestion: File already exists]
[2025-03-16 19:16:25,215: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-03-16 19:16:25,215: INFO: main: >>>> stage Data Validation Stage started <<<<]
[2025-03-16 19:16:25,218: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 19:16:25,224: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 19:16:25,226: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 19:16:25,226: INFO: common: created directory at: artifacts]
[2025-03-16 19:16:25,230: INFO: common: created directory at: artifacts/data_validation]
[2025-03-16 19:16:25,255: INFO: main: >>>> stage Data Validation Stage completed <<<<

x======x]
[2025-03-16 19:16:25,255: INFO: main: >>> stage Data Transformation stage started <<<]
[2025-03-16 19:16:25,270: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 19:16:25,273: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 19:16:25,276: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 19:16:25,278: INFO: common: created directory at: artifacts]
[2025-03-16 19:16:25,278: INFO: common: created directory at: artifacts/data_transformation]
[2025-03-16 19:16:25,303: INFO: data_transformation: Splitted data into training and test sets]
[2025-03-16 19:16:25,303: INFO: data_transformation: (1199, 12)]
[2025-03-16 19:16:25,303: INFO: data_transformation: (400, 12)]
[2025-03-16 19:16:25,303: INFO: main: >>> stage Data Transformation stage completed <<<

x=====x]
[2025-03-16 19:16:25,303: INFO: main: >>> stage Model Trainer Stage started <<<]
[2025-03-16 19:16:25,309: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 19:16:25,312: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 19:16:25,315: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 19:16:25,317: INFO: common: created directory at: artifacts]
[2025-03-16 19:16:25,317: INFO: common: created directory at: artifacts/model_trainer]
[2025-03-16 19:16:25,362: INFO: main: >>> stage Model Trainer Stage completed <<<

x=====x]
[2025-03-16 19:16:25,362: INFO: main: >>> stage Model evaluation Stage started <<<]
[2025-03-16 19:16:25,362: ERROR: main: 'ModelTrainerTrainingPipeline' object has no attribute 'initiate_model_evaluation']
Traceback (most recent call last):
  File "E:\DataScienceProject1\main.py", line 58, in <module>
    model_trainer.initiate_model_evaluation()
AttributeError: 'ModelTrainerTrainingPipeline' object has no attribute 'initiate_model_evaluation'
[2025-03-16 19:19:03,761: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-03-16 19:19:03,761: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 19:19:03,761: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 19:19:03,761: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 19:19:03,761: INFO: common: created directory at: artifacts]
[2025-03-16 19:19:03,761: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-16 19:19:03,761: INFO: data_ingestion: File already exists]
[2025-03-16 19:19:03,761: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-03-16 19:19:03,761: INFO: main: >>>> stage Data Validation Stage started <<<<]
[2025-03-16 19:19:03,784: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 19:19:03,790: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 19:19:03,794: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 19:19:03,794: INFO: common: created directory at: artifacts]
[2025-03-16 19:19:03,794: INFO: common: created directory at: artifacts/data_validation]
[2025-03-16 19:19:03,824: INFO: main: >>>> stage Data Validation Stage completed <<<<

x======x]
[2025-03-16 19:19:03,825: INFO: main: >>> stage Data Transformation stage started <<<]
[2025-03-16 19:19:03,826: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 19:19:03,826: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 19:19:03,834: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 19:19:03,834: INFO: common: created directory at: artifacts]
[2025-03-16 19:19:03,834: INFO: common: created directory at: artifacts/data_transformation]
[2025-03-16 19:19:03,869: INFO: data_transformation: Splitted data into training and test sets]
[2025-03-16 19:19:03,869: INFO: data_transformation: (1199, 12)]
[2025-03-16 19:19:03,869: INFO: data_transformation: (400, 12)]
[2025-03-16 19:19:03,871: INFO: main: >>> stage Data Transformation stage completed <<<

x=====x]
[2025-03-16 19:19:03,871: INFO: main: >>> stage Model Trainer Stage started <<<]
[2025-03-16 19:19:03,876: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 19:19:03,876: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 19:19:03,880: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 19:19:03,882: INFO: common: created directory at: artifacts]
[2025-03-16 19:19:03,884: INFO: common: created directory at: artifacts/model_trainer]
[2025-03-16 19:19:03,929: INFO: main: >>> stage Model Trainer Stage completed <<<

x=====x]
[2025-03-16 19:19:03,931: INFO: main: >>> stage Model evaluation Stage started <<<]
[2025-03-16 19:19:03,933: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 19:19:03,936: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 19:19:03,938: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 19:19:03,940: INFO: common: created directory at: artifacts]
[2025-03-16 19:19:03,941: INFO: common: created directory at: artifacts/model_evaluation]
[2025-03-16 19:19:05,628: ERROR: main: name 'save_json' is not defined]
Traceback (most recent call last):
  File "E:\DataScienceProject1\main.py", line 58, in <module>
    model_evaluation.initiate_model_evaluation()
  File "E:\DataScienceProject1\src\datascience\pipeline\model_evaluation_pipeline.py", line 15, in initiate_model_evaluation
    model_evaluation.log_into_mlflow()
  File "E:\DataScienceProject1\src\datascience\components\model_evaluation.py", line 42, in log_into_mlflow
    save_json(path=Path(self.config.metric_file_name), data=scores)
NameError: name 'save_json' is not defined
[2025-03-16 19:29:09,734: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-03-16 19:29:09,736: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 19:29:09,736: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 19:29:09,742: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 19:29:09,742: INFO: common: created directory at: artifacts]
[2025-03-16 19:29:09,744: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-16 19:29:09,744: INFO: data_ingestion: File already exists]
[2025-03-16 19:29:09,748: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-03-16 19:29:09,748: INFO: main: >>>> stage Data Validation Stage started <<<<]
[2025-03-16 19:29:09,756: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 19:29:09,756: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 19:29:09,761: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 19:29:09,761: INFO: common: created directory at: artifacts]
[2025-03-16 19:29:09,765: INFO: common: created directory at: artifacts/data_validation]
[2025-03-16 19:29:09,793: INFO: main: >>>> stage Data Validation Stage completed <<<<

x======x]
[2025-03-16 19:29:09,793: INFO: main: >>> stage Data Transformation stage started <<<]
[2025-03-16 19:29:09,809: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 19:29:09,810: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 19:29:09,813: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 19:29:09,813: INFO: common: created directory at: artifacts]
[2025-03-16 19:29:09,815: INFO: common: created directory at: artifacts/data_transformation]
[2025-03-16 19:29:09,840: INFO: data_transformation: Splitted data into training and test sets]
[2025-03-16 19:29:09,841: INFO: data_transformation: (1199, 12)]
[2025-03-16 19:29:09,841: INFO: data_transformation: (400, 12)]
[2025-03-16 19:29:09,841: INFO: main: >>> stage Data Transformation stage completed <<<

x=====x]
[2025-03-16 19:29:09,841: INFO: main: >>> stage Model Trainer Stage started <<<]
[2025-03-16 19:29:09,846: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 19:29:09,846: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 19:29:09,851: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 19:29:09,851: INFO: common: created directory at: artifacts]
[2025-03-16 19:29:09,851: INFO: common: created directory at: artifacts/model_trainer]
[2025-03-16 19:29:09,900: INFO: main: >>> stage Model Trainer Stage completed <<<

x=====x]
[2025-03-16 19:29:09,902: INFO: main: >>> stage Model evaluation Stage started <<<]
[2025-03-16 19:29:09,903: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 19:29:09,903: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 19:29:09,910: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 19:29:09,911: INFO: common: created directory at: artifacts]
[2025-03-16 19:29:09,911: INFO: common: created directory at: artifacts/model_evaluation]
[2025-03-16 19:29:11,080: ERROR: main: name 'save_json' is not defined]
Traceback (most recent call last):
  File "E:\DataScienceProject1\main.py", line 58, in <module>
    model_evaluation.initiate_model_evaluation()
  File "E:\DataScienceProject1\src\datascience\pipeline\model_evaluation_pipeline.py", line 15, in initiate_model_evaluation
    model_evaluation.log_into_mlflow()
  File "E:\DataScienceProject1\src\datascience\components\model_evaluation.py", line 44, in log_into_mlflow
    save_json(path=Path(self.config.metric_file_name), data=scores)
NameError: name 'save_json' is not defined
[2025-03-16 19:35:39,738: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-03-16 19:35:39,741: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 19:35:39,741: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 19:35:39,747: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 19:35:39,747: INFO: common: created directory at: artifacts]
[2025-03-16 19:35:39,747: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-16 19:35:39,747: INFO: data_ingestion: File already exists]
[2025-03-16 19:35:39,750: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-03-16 19:35:39,750: INFO: main: >>>> stage Data Validation Stage started <<<<]
[2025-03-16 19:35:39,756: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 19:35:39,759: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 19:35:39,764: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 19:35:39,766: INFO: common: created directory at: artifacts]
[2025-03-16 19:35:39,766: INFO: common: created directory at: artifacts/data_validation]
[2025-03-16 19:35:39,794: INFO: main: >>>> stage Data Validation Stage completed <<<<

x======x]
[2025-03-16 19:35:39,795: INFO: main: >>> stage Data Transformation stage started <<<]
[2025-03-16 19:35:39,799: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 19:35:39,801: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 19:35:39,805: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 19:35:39,807: INFO: common: created directory at: artifacts]
[2025-03-16 19:35:39,807: INFO: common: created directory at: artifacts/data_transformation]
[2025-03-16 19:35:39,834: INFO: data_transformation: Splitted data into training and test sets]
[2025-03-16 19:35:39,834: INFO: data_transformation: (1199, 12)]
[2025-03-16 19:35:39,837: INFO: data_transformation: (400, 12)]
[2025-03-16 19:35:39,837: INFO: main: >>> stage Data Transformation stage completed <<<

x=====x]
[2025-03-16 19:35:39,837: INFO: main: >>> stage Model Trainer Stage started <<<]
[2025-03-16 19:35:39,841: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 19:35:39,841: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 19:35:39,845: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 19:35:39,847: INFO: common: created directory at: artifacts]
[2025-03-16 19:35:39,848: INFO: common: created directory at: artifacts/model_trainer]
[2025-03-16 19:35:39,889: INFO: main: >>> stage Model Trainer Stage completed <<<

x=====x]
[2025-03-16 19:35:39,889: INFO: main: >>> stage Model evaluation Stage started <<<]
[2025-03-16 19:35:39,893: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 19:35:39,897: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 19:35:39,900: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 19:35:39,901: INFO: common: created directory at: artifacts]
[2025-03-16 19:35:39,902: INFO: common: created directory at: artifacts/model_evaluation]
[2025-03-16 19:35:41,105: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json]
[2025-03-16 19:35:50,603: INFO: main: >>> stage Model evaluation Stage completed <<<

x=====x]
[2025-03-16 19:39:37,919: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-03-16 19:39:37,923: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 19:39:37,923: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 19:39:37,928: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 19:39:37,928: INFO: common: created directory at: artifacts]
[2025-03-16 19:39:37,928: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-16 19:39:37,928: INFO: data_ingestion: File already exists]
[2025-03-16 19:39:37,932: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-03-16 19:39:37,934: INFO: main: >>>> stage Data Validation Stage started <<<<]
[2025-03-16 19:39:37,943: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 19:39:37,946: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 19:39:37,946: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 19:39:37,946: INFO: common: created directory at: artifacts]
[2025-03-16 19:39:37,946: INFO: common: created directory at: artifacts/data_validation]
[2025-03-16 19:39:37,977: INFO: main: >>>> stage Data Validation Stage completed <<<<

x======x]
[2025-03-16 19:39:37,977: INFO: main: >>> stage Data Transformation stage started <<<]
[2025-03-16 19:39:37,981: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 19:39:37,987: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 19:39:37,989: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 19:39:37,989: INFO: common: created directory at: artifacts]
[2025-03-16 19:39:37,989: INFO: common: created directory at: artifacts/data_transformation]
[2025-03-16 19:39:38,016: INFO: data_transformation: Splitted data into training and test sets]
[2025-03-16 19:39:38,016: INFO: data_transformation: (1199, 12)]
[2025-03-16 19:39:38,016: INFO: data_transformation: (400, 12)]
[2025-03-16 19:39:38,016: INFO: main: >>> stage Data Transformation stage completed <<<

x=====x]
[2025-03-16 19:39:38,016: INFO: main: >>> stage Model Trainer Stage started <<<]
[2025-03-16 19:39:38,022: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 19:39:38,024: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 19:39:38,025: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 19:39:38,025: INFO: common: created directory at: artifacts]
[2025-03-16 19:39:38,029: INFO: common: created directory at: artifacts/model_trainer]
[2025-03-16 19:39:38,081: INFO: main: >>> stage Model Trainer Stage completed <<<

x=====x]
[2025-03-16 19:39:38,081: INFO: main: >>> stage Model evaluation Stage started <<<]
[2025-03-16 19:39:38,082: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 19:39:38,082: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 19:39:38,088: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 19:39:38,089: INFO: common: created directory at: artifacts]
[2025-03-16 19:39:38,089: INFO: common: created directory at: artifacts/model_evaluation]
[2025-03-16 19:39:39,721: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json]
[2025-03-16 19:39:56,400: INFO: main: >>> stage Model evaluation Stage completed <<<

x=====x]
[2025-03-16 20:39:38,285: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.0.101:8080]
[2025-03-16 20:39:38,285: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-03-16 20:39:58,782: INFO: _internal: 127.0.0.1 - - [16/Mar/2025 20:39:58] "GET / HTTP/1.1" 200 -]
[2025-03-16 20:39:59,099: INFO: _internal: 127.0.0.1 - - [16/Mar/2025 20:39:59] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2025-03-16 20:39:59,136: INFO: _internal: 127.0.0.1 - - [16/Mar/2025 20:39:59] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2025-03-16 20:40:41,838: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-03-16 20:40:41,844: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 20:40:41,844: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 20:40:41,847: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 20:40:41,847: INFO: common: created directory at: artifacts]
[2025-03-16 20:40:41,847: INFO: common: created directory at: artifacts/data_ingestion]
[2025-03-16 20:40:41,851: INFO: data_ingestion: File already exists]
[2025-03-16 20:40:41,855: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-03-16 20:40:41,855: INFO: main: >>>> stage Data Validation Stage started <<<<]
[2025-03-16 20:40:41,860: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 20:40:41,861: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 20:40:41,865: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 20:40:41,865: INFO: common: created directory at: artifacts]
[2025-03-16 20:40:41,865: INFO: common: created directory at: artifacts/data_validation]
[2025-03-16 20:40:41,896: INFO: main: >>>> stage Data Validation Stage completed <<<<

x======x]
[2025-03-16 20:40:41,896: INFO: main: >>> stage Data Transformation stage started <<<]
[2025-03-16 20:40:41,901: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 20:40:41,905: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 20:40:41,908: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 20:40:41,908: INFO: common: created directory at: artifacts]
[2025-03-16 20:40:41,909: INFO: common: created directory at: artifacts/data_transformation]
[2025-03-16 20:40:41,933: INFO: data_transformation: Splitted data into training and test sets]
[2025-03-16 20:40:41,933: INFO: data_transformation: (1199, 12)]
[2025-03-16 20:40:41,933: INFO: data_transformation: (400, 12)]
[2025-03-16 20:40:41,933: INFO: main: >>> stage Data Transformation stage completed <<<

x=====x]
[2025-03-16 20:40:41,933: INFO: main: >>> stage Model Trainer Stage started <<<]
[2025-03-16 20:40:41,941: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 20:40:41,941: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 20:40:41,945: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 20:40:41,946: INFO: common: created directory at: artifacts]
[2025-03-16 20:40:41,946: INFO: common: created directory at: artifacts/model_trainer]
[2025-03-16 20:40:41,991: INFO: main: >>> stage Model Trainer Stage completed <<<

x=====x]
[2025-03-16 20:40:41,991: INFO: main: >>> stage Model evaluation Stage started <<<]
[2025-03-16 20:40:41,996: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-03-16 20:40:41,996: INFO: common: yaml file: params.yaml loaded successfully]
[2025-03-16 20:40:42,003: INFO: common: yaml file: schema.yaml loaded successfully]
[2025-03-16 20:40:42,003: INFO: common: created directory at: artifacts]
[2025-03-16 20:40:42,003: INFO: common: created directory at: artifacts/model_evaluation]
[2025-03-16 20:40:43,514: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json]
[2025-03-16 20:40:52,312: INFO: main: >>> stage Model evaluation Stage completed <<<

x=====x]
[2025-03-16 20:40:53,056: INFO: _internal: 127.0.0.1 - - [16/Mar/2025 20:40:53] "GET /train HTTP/1.1" 200 -]
[2025-03-16 20:41:26,311: INFO: _internal: 127.0.0.1 - - [16/Mar/2025 20:41:26] "GET / HTTP/1.1" 200 -]
[2025-03-16 20:41:26,345: INFO: _internal: 127.0.0.1 - - [16/Mar/2025 20:41:26] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2025-03-16 20:42:31,378: INFO: _internal: 127.0.0.1 - - [16/Mar/2025 20:42:31] "POST /predict HTTP/1.1" 200 -]
[2025-03-16 20:42:37,075: INFO: _internal: 127.0.0.1 - - [16/Mar/2025 20:42:37] "GET / HTTP/1.1" 200 -]
[2025-03-16 20:42:37,108: INFO: _internal: 127.0.0.1 - - [16/Mar/2025 20:42:37] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2025-03-16 20:42:53,805: INFO: _internal: 127.0.0.1 - - [16/Mar/2025 20:42:53] "POST /predict HTTP/1.1" 200 -]
[2025-03-16 20:46:38,167: INFO: _internal: 127.0.0.1 - - [16/Mar/2025 20:46:38] "GET / HTTP/1.1" 200 -]
[2025-03-16 20:46:38,820: INFO: _internal: 127.0.0.1 - - [16/Mar/2025 20:46:38] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2025-03-16 20:46:38,911: INFO: _internal: 127.0.0.1 - - [16/Mar/2025 20:46:38] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2025-03-16 20:46:51,849: INFO: _internal: 127.0.0.1 - - [16/Mar/2025 20:46:51] "POST /predict HTTP/1.1" 200 -]
[2025-03-16 20:49:11,649: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.0.101:8080]
[2025-03-16 20:49:11,649: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-03-16 20:49:16,199: INFO: _internal: 127.0.0.1 - - [16/Mar/2025 20:49:16] "GET / HTTP/1.1" 200 -]
[2025-03-16 20:49:16,389: INFO: _internal: 127.0.0.1 - - [16/Mar/2025 20:49:16] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2025-03-16 20:49:28,136: INFO: _internal: 127.0.0.1 - - [16/Mar/2025 20:49:28] "POST /predict HTTP/1.1" 200 -]
[2025-03-16 20:52:56,028: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.0.101:8080]
[2025-03-16 20:52:56,030: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2025-03-16 20:53:00,812: INFO: _internal: 127.0.0.1 - - [16/Mar/2025 20:53:00] "GET / HTTP/1.1" 200 -]
[2025-03-16 20:53:01,198: INFO: _internal: 127.0.0.1 - - [16/Mar/2025 20:53:01] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2025-03-16 20:53:01,238: INFO: _internal: 127.0.0.1 - - [16/Mar/2025 20:53:01] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2025-03-16 20:53:11,110: INFO: _internal: 127.0.0.1 - - [16/Mar/2025 20:53:11] "POST /predict HTTP/1.1" 200 -]
